helm-charts/values.yaml

```diff
@@ -261,19 +268,6 @@ wso2:
               limits:
                 memory: "1028Mi"
                 cpu: "1000m"
-            env:
```

Comment:
are we removing these env since they are defaulted values?

Reply:
yes